tests/chainer_tests/test_link.py

```diff
@@ -649,6 +650,51 @@ def test_to_chainerx(self, backend_config):
         self.assertEqual(self.link._device, expected_device)
 
 
+class TestLinkMissingInitCall(unittest.TestCase):
+    # Tests for detecting incorrectly written Link subclasses in which
+    # the call to Link.__init__ is missing
+
+    expected_message = 'Link.__init__() has not been called.'
+
+    def test_missing1(self):
+        # Nothing is done in __init__.
+        # The fault should be detected no later than __call__().
+
+        class Derived(chainer.Link):
+            def __init__(self):
+                pass
+
+            def forward(self, x):
+                return x
+
+        with pytest.raises(RuntimeError, message=self.expected_message):
```

Comment:
It looks `match` is the correct option to use instead of `message` (the latter just prints the message when the test fails). CI failed due to the warning.